Stripe/Checkout/STPCheckoutOptions.m

```diff
@@ -67,6 +67,23 @@ - (NSString *)stringifiedJSONRepresentation {
     return [[NSString alloc] initWithData:[NSJSONSerialization dataWithJSONObject:values options:0 error:nil] encoding:NSUTF8StringEncoding];
 }
 
+- (void)setLogoImage:(STP_IMAGE_CLASS * __stp_nullable)logoImage {
```

Comment:
This has to do with how objective-c `@property`s work. Let's say in a header you declare that a class has, say, `@property NSString *title`. This will implicitly create an instance variable named `_title`, as well as setters and getters of the format `- (void)setTitle:(NSString *)title` and `- (NSString *)title`.

You can optionally override those setters and getters (as I've done here) to make them have additional side effects, or change how the information is stored, or all kinds of other weird things. These methods will be called under the hood when you use dot-syntax to access or modify that property. So if elsewhere in my code I call `checkoutOptions.logoImage = myImage`, it's equivalent to writing `[checkoutOptions setLogoImage:myImage]`.

Properties are capable of doing way more work for you - you can see some additional qualifiers (e.g. `strong`/`weak`, `nonatomic`/`atomic`, `readwrite`/`readonly`, and as of recently `nonnull`/`nullable`) elsewhere in the source.

If this wasn't already too much detail, you can learn more at https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html